keylime/tpm/tpm_abstract.py

```diff
@@ -115,23 +124,23 @@ def warn_emulator(self):
             logger.warning("INSECURE: Only use Keylime in this mode for testing or debugging purposes.")
 
     @staticmethod
-    def __read_tpm_data():
+    def __read_tpm_data() -> Dict:
```

Comment:
Should be needed to add the projection types?

Reply:
`def __read_tpm_data() -> Dict[str, bytes]:`

Reply:
Thanks.